src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,41 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  display: flex;
+  flex: 0 0 auto;
+  flex-direction: column;
+  height: 100%;
+  min-width: $iui-3xl * 2;
+  width: $iui-3xl * 4;
+  max-width: 50vw;
+  box-sizing: border-box;
+  padding: $iui-baseline $iui-sm;
+  overflow-x: hidden;
+  overflow-y: overlay;
+  resize: horizontal;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: max-width $iui-speed-fast ease-out;
+  }
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-right: 1px solid t(iui-color-background-5);
+  }
+
+  .iui-side-navigation-title {
```

Comment:
I am not sure about this class and whether we need to hide it on expand. I thought users will use our typography `Title` component. Also, here is even an example with more than just a title:
![image](https://user-images.githubusercontent.com/36186912/132340816-ad90b9ed-ad45-4da1-8501-cde914abfee7.png)

Reply:
I'd like to remove the title because it is repeated in the expanded side navigation and can free up some space.  However, it makes it very challenging when figuring out where to place the settings button.  Because of this I have removed CSS that hides the submenu title when the side navigation is expanded.